.changeset/tame-numbers-smile.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-kubernetes-backend': minor
+'@backstage/plugin-kubernetes-node': minor
```

Comment:
since we're pre-1.0 and these are backwards-compatible
```suggestion
'@backstage/plugin-kubernetes-backend': patch
'@backstage/plugin-kubernetes-node': patch
```

Reply:
Sure !